packages/tfchain_client/src/contracts.ts

```diff
@@ -12,6 +12,16 @@ interface NameContract {
   name: string;
 }
 
+export interface ContractLockOptions {
+  id: number;
+}
+
+export interface ContractLock {
+  amountLocked: number;
+  lockUpdated: number;

```

Comment:
what does this field do?

Reply:
all these fields come from the chain, I just wrapped them

for the lock update, this is a timestamp field, I didn't use it in any place